test/Website/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/dotnet:1.0-runtime
```

Comment:
Yes, I was aware of 1.1.0, but opted for 1.0.x here as the test project specifies `netcoreapp1.0` as the runtime, and therefore was not able to run on `1.1.0`. Given this is the test project though I suppose there would be no harm in using the latest runtime here. It will also be a good test that the library is capable of running on newer versions of the runtime. Agree that making it immutable is desirable too.